test/test_dataloader.py

```diff
@@ -283,11 +283,11 @@ def test_iterable_dataset_err(self):
 # takes in dummy var so this can also be used as a `worker_init_fn`
 def set_faulthander_if_available(_=None):
     if HAS_FAULTHANDLER:
-        faulthandler.enable()
+        faulthandler.enable(sys.__stderr__)
```

Comment:
`xml-test-runner` takes control of the default `sys.stderr`, and their proxy version doesn't have `fileno()` defined, which causes this line to fail. This change just has the fault handler always spit out tracebacks to the "real" `stderr` instead of a proxy.